lib/uplinkc/uplink_definitions.h

```diff
@@ -51,6 +51,27 @@ typedef struct EncryptionParameters {
     int32_t     block_size;
 } EncryptionParameters;
 
+
+typedef struct EncryptionRestriction {
+    char *bucket;
+    char *path_prefix;
+} EncryptionRestriction;
+
+typedef struct CaveatPath {
```

Comment:
remove `CaveatPath` because we won't be using them for now.

Reply:
Removed CaveatPath for now